src/mock/film.js

```diff
@@ -36,32 +53,38 @@ const DIRECTORS = [`Petya Ivanov`, `Vasiliy Pupkin`, `Roman Chepuha`];
 const WRITERS = [`Anne Wigton`, `Heinz Herald`, `Richard Weil`];
 const RELEASE_DATE = [`30 March`, `11 March`, `10 March`];
 const COUNTRYS = [`USA`, `USSR`, `Brazil`];
+const AGES = [18, 16, 6];
 
-// console.log(getRandomElementsArray(DESCRIPTIONS));
 function createMockFilmCard() {
   const item = {
     name: getRandomArrayElement(FILMS_NAMES),
+    originalName: getRandomArrayElement(FILMS_NAMES),
     rating: getRandomInteger(0, 10),
     poster: getRandomArrayElement(POSTERS),
     comments: Array.from(new Set(getRandomElementsArray(COMMENTS_ID))),
     yearProduction: getRandomArrayElement(YEAR_PRODUCTION),
     duration: getRandomArrayElement(DURATION),
-    genre: getRandomElementsArray(GENRE),
+    genres: getRandomElementsArray(GENRE),
     description: getRandomElementsArray(DESCRIPTIONS).join(`,`),
     director: getRandomArrayElement(DIRECTORS),
-    writerts: getRandomElementsArray(WRITERS),
+    writers: getRandomElementsArray(WRITERS),
     actors: getRandomElementsArray(WRITERS),
     country: getRandomArrayElement(COUNTRYS),
     isInWatchList: Boolean(getRandomInteger(0, 1)),
     isInHistory: Boolean(getRandomInteger(0, 1)),
-    isInFavorite: Boolean(getRandomInteger(0, 1))
+    isInFavorite: Boolean(getRandomInteger(0, 1)),
+    age: getRandomArrayElement(AGES)
```

Comment:
я бы назвал более специфично:
```suggestion
    minAge: getRandomArrayElement(AGES)
```